src/components/IKImage/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useContext, useEffect, useRef, useState } from "react";
+import NextImage, { ImageProps } from "next/image";
+import IKImageProps from "./props";
+import ImageKitProviderProps from "../ImageKitProvider/props";
+import useImageKitComponent from "../ImageKitComponent";
+import { ImageKitContext } from "../ImageKitProvider";
+import { fetchEffectiveConnection, getIKElementsUrl, getSrc } from "../../utils/Utility";
+
+const IKImage = (props: Omit<ImageProps, "src" | "loading"> & IKImageProps & ImageKitProviderProps) => {
```

Comment:
![image](https://github.com/imagekit-developer/imagekit-next/assets/8640877/6076b07a-9869-428a-b526-a64301977275)
Getting this error when doing fill={true}
```
<IKImage path="/default-image.jpg" fill={true} alt="Alt text"/>
```

Reply:
done